docs/methods/preprocessing/label_encoder.md

```diff
@@ -0,0 +1,19 @@
+# Label Encoder
+
+Encode target labels with value between 0 and n_classes-1.
+
+This transformer should be used to encode target values, i.e. y, and not the input X.
+
+## Parameters
+
+| Name         | Definition                                 | Type   |
+|--------------|--------------------------------------------|--------|
+| targetVector | The target labels which need to be encoded | Vector |
```

Comment:
Write vector between tilde and in lowercase along with the type like this `vector<int>`

Reply:
changed sir